packages/presentation/lib/screens/home/home_screen.dart

```diff
@@ -12,11 +22,286 @@ class HomeScreen extends StatelessWidget {
+                                ),
+                              const SizedBox(
+                                width: AppSizes.size6,
+                              ),
+                              Text(
+                                'Coming Soon',
+                                style: selectedType ==
+                                        SelectedMoviesType.comingSoon
+                                    ? SelectionButtonStyles
+                                        .activeButtonTextStyle
+                                    : SelectionButtonStyles
+                                        .inactiveButtonTextStyle,
+                              ),
+                            ],
+                          ),
+                        ),
+                      ),
+                    ),
+                  ),
+                ],
+              ),
+            ),
+            const SizedBox(
+              height: AppSizes.size24,
+            ),
+            StreamBuilder(
+              stream: bloc.stream,
+              builder: (context, snapshot) {
+                final data = snapshot.data;
+                if (data != null) {
+                  return Expanded(
+                    child: GridView.builder(
+                      shrinkWrap: true,
+                      itemCount: data.movies.length,
+                      gridDelegate:
+                          const SliverGridDelegateWithFixedCrossAxisCount(
+                        crossAxisCount: 2,
+                        childAspectRatio: HomeScreenSizes.gridChildAspecRatio,
+                        crossAxisSpacing: AppSizes.size13,
+                        mainAxisSpacing:
+                            HomeScreenSizes.gridViewMainAxisSpacing,
+                      ),
+                      itemBuilder: (context, index) {
+                        final movie = data.movies[index];
+
+                        return GestureDetector(
+                          behavior: HitTestBehavior.translucent,
+                          onTap: () {
+                            bloc.goToMovieDetailsPage(movie);
+                          },
+                          child: SingleChildScrollView(
+                            child: MovieCard(
+                              movie.title,
+                              rating: movie.rating,
+                              genres: movie.genres,
+                              runtime: bloc.convertApiRuntime(movie.runtime),
+                              certification: movie.certification,
+                              imageUrl: bloc.getImageUrlById(movie.imdbId),
+                            ),
+                          ),
+                        );
+                      },
+                    ),
+                  );
+                } else {
+                  return const ShimmerLoader();
+                }
+              },
+            ),
+          ],
+        ),
+      ),
+      bottomNavigationBar: Container(
```

Comment:
placing bottom navigation into home screen not the best solution. why? in this case u have to add bottom navigation to each screen, while it should be independent from the screen and have it's own state. let's move it to App level. we also have scaffold here

Reply:
Fixed